templates/seaice/point.html

```diff
@@ -0,0 +1,51 @@
+{% extends 'base.html' %} {% block content %}
+<h3>Pan&ndash;Arctic Sea Ice Concentration Point Query</h3>
+<p>
+  This endpoint accesses historical monthly data on sea ice concentration in the
+    Arctic from 1850-2021.
```

Comment:
So so minor, but ideally this would be `&ndash;` ... only worth changing to get in the habit of looking at dashes and stuff with a careful eye when writing anything in HTML!